src/components/atoms/MultiSelect/Select/Select.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react';
+import { ArrowIcon } from 'assets/index';
+import {
+  StyledArrowImage,
+  StyledSelect,
+  StyledSelectCaption,
+  StyledTopWrapper,
+  StyledWrapper,
+} from './Select.styled';
+import Option from '../Option';
+import { ISelect, ISingleOption } from './Select.model';
+
+const Select = ({ isMulti, handleOnClick, options }: ISelect): JSX.Element => {
+  const [isSelectOpened, setIsSelectOpened] = useState<boolean>(false);
+  const [selectedOptions, setSelectedOptions] = useState<ISingleOption[]>([]);
+  const [currentOptionValue, setCurrentOptionValue] = useState<string>('');
+
+  const doesArrayConsistValue = (value: string) => {
+    return selectedOptions.filter((element) => element.value === value).length;
```

Comment:
Enter the type of element

Reply:
Is it neccessary if TS doesn't throw any errors ?

Reply:
In the morning I saw Error with this logic but now I don't have. Nvm.